integrationtest/java/modules/orm-lucene/src/test/java/org/hibernate/search/integrationtest/java/modules/orm/lucene/service/JavaModulePathIT.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.integrationtest.java.modules.orm.lucene.service;
+
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import org.hibernate.search.mapper.orm.Search;
+
+import org.junit.Test;
+
+public class JavaModulePathIT {
+
+	/*
+	 * Test that the service successfully uses Hibernate Search in the module path.
+	 * We don't really care about the features themselves,
+	 * but the easiest way to check this is to just use Hibernate Search features and see if it works.
+	 */
+	@Test
+	public void test() {
+		checkIsInModulePath( Object.class );
+		checkIsInModulePath( AuthorService.class );
+		checkIsInModulePath( Search.class );
+
+		AuthorService service = new AuthorService();
+		service.add( "foo" );
+		service.add( "bar" );
+		service.add( "foo bar" );
+		assertEquals( 2, service.search( "foo" ).size() );
+
+		//TODO: uncomment out and figure out what's not working:
+//		assertThatThrownBy( service::triggerValidationFailure )
+//				.isInstanceOf( SearchException.class )
+//				.hasMessageContainingAll(
+//						"Hibernate Search encountered failures during Schema management",
+//						"attribute 'analyzer'",
+//						"Invalid value. Expected 'default', actual is 'myAnalyzer'"
+//				);
```

Comment:
Lucene schema validation actually doesn't do much, and indeed it never checks analyzers: https://docs.jboss.org/hibernate/stable/search/reference/en-US/html_single/#mapper-orm-schema-management-concepts-lucene-schema

So yes, you can drop this and all code related to the schema validation failure (`AuthorWithSlightlyDifferentIndex`, etc.) in this module.